src/database.rs

```diff
@@ -20,8 +20,13 @@ pub fn make_db_pool() -> Result<DbPool> {
         .parse::<u32>()
         .context("Invalid value provided for \"DB_MAX_CONNECTIONS\" expected numeric value")?;
 
+    let connection_timeout = match var("POSTGRES_CONN_TIMEOUT") {
+        Ok(timeout) => timeout.parse().unwrap_or(2),
+        Err(_) => 2,
+    };
```

Comment:
This looks like a nice use case for `unwrap_or_else`:

```suggestion
    let connection_timeout = var("POSTGRES_CONN_TIMEOUT")
        .unwrap_or_else(|_| String::from("2"))
        .parse:<u32>()
        .context("The value provided to \"POSTGRES_CONN_TIMEOUT\" is not numeric")?;
```